tests/integration/test_integration.py

```diff
@@ -1437,7 +1369,9 @@ def req(
             EXAMPLE_PYTHON_REQUIREMENTS_URL,
             fetcher=URLFetcher(
                 NetworkConfiguration(
-                    proxy="{proxy_auth}@localhost:{port}".format(proxy_auth=proxy_auth, port=port),
+                    proxy="http://{proxy_auth}@localhost:{port}".format(
```

Comment:
This was just to prove to myself that Fetcher / urllib could work with the protocol-prfixed form. Pip, it turns out, requires that unlike Fetcher / urllib; so FAPP users must pass using `--proxy http(s)://...` syntax.

Reply:
Re-confirmed direct use of the pip CLI, whose --help lies here:
```
$ PYTHONPATH=pex/vendor/_vendored/pip python -mpip -h
...
General Options:
  -h, --help                  Show help.
...
  --proxy <proxy>             Specify a proxy in the form [user:passwd@]proxy.server:port.
```
also requires `http(s)://` or else fails with:
```
$ PYTHONPATH=pex/vendor/_vendored/pip python -mpip --proxy localhost:8888 download --no-deps ansicolors --dest wheels
...
  File "/home/jsirois/dev/pantsbuild/jsirois-pex/pex/vendor/_vendored/pip/pip/_vendor/requests/adapters.py", line 414, in send
    raise InvalidURL(e, request=request)
pip._vendor.requests.exceptions.InvalidURL: Not supported proxy scheme localhost
```

Reply:
Fixed Pex --proxy help to not parrot Pip --help.